src/components/VerticalListMenu/VerticalListMenu.jsx

```diff
@@ -0,0 +1,209 @@
+} = React.PropTypes;
+
+/**
+ * {"categories": ["layout"], "madeFrom": ["ChevronIcon"]}
+ *
+ * Used primarily for navigation lists. It supports nesting `VerticalListMenu`s
+ * below `VerticalListMenu.Item`s and animating expanding of those sub lists.
+ * The default reducer behavior is for only one `VerticalListMenu.Item` to be
+ * selected at any given time; that is easily overridden by handling `onSelect`
+ * yourself.
+ */
+const VerticalListMenu = React.createClass(createLucidComponentDefinition({
+	displayName: 'VerticalListMenu',
+
+	reducers,
+
+	childProps: {
+		Item: {
+			hasExpander: bool,
+			isExpanded: bool,
+			isSelected: bool,
+			onSelect: func,
+			onToggle: func,
+		}
+	},
+
+	propTypes: {
+		/**
+		 * Regular `children` aren't really used in this component, but if you do
+		 * add them they will be placed at the end of the component. You should be
+		 * using `VerticalListMenu.Item`s instead of regular children.
+		 */
+		children: node,
+
+		/**
+		 * Appended to the component-specific class names set on the root element.
+		 */
+		className: string,
+
+		/**
+		 * Passed through to the root element.
+		 */
+		style: object,
+
+		/**
+		 * Indicates which of the `VerticalListMenu.Item` children are currently
+		 * selected. You can also put the `isSelected` prop directly on the
+		 * `VerticalListMenu.Item`s if you wish.
+		 */
+		selectedIndices: arrayOf(number),
+
+		/**
+		 * Indicates which of the `VerticalListMenu.Item` children are currently
+		 * expanded. You can also put the `isExpanded` prop directly on the
+		 * `VerticalListMenu.Item`s if you wish.
+		 */
+		expandedIndices: arrayOf(number),
+
+		/**
+		 * Callback fired when the user selects a `VerticalListMenu.Item`.
+		 *
+		 * Signature: `(index, { event, props }) => {}`
+		 */
+		onSelect: func,
+
+		/**
+		 * Callback fired when the user expands or collapses an expandable `VerticalListMenu.Item`.
+		 *
+		 * Signature: `(index, { event, props }) => {}`
+		 */
+		onToggle: func,
+	},
+
+	getDefaultProps() {
+		return {
+			onSelect: _.noop,
+			onToggle: _.noop,
+			expandedIndices: [],
+			selectedIndices: [],
+		};
+	},
+
+	render() {
+		const {
+			children,
+			className,
+			style,
+			selectedIndices,
+			expandedIndices,
+			...passThroughs
+		} = this.props;
+
+		const itemChildProps = VerticalListMenu.Item.findInAllAsProps(this.props);
+
+		return (
+			<ul
+				{...passThroughs}
+				className={boundClassNames('&', className)}
+				style={style}
+			>
+				{_.map(itemChildProps, (itemChildProp, index) => {
+					const {
+						hasExpander = false,
+					} = itemChildProp;
+
+					const itemChildrenAsArray = React.Children.toArray(itemChildProp.children);
+
+					// Was not able to get `child.Type` to work correctly, I suspect this
+					// is due to the way we wrap components with createLucidComponentDefinition
+					const listChildren = _.filter(itemChildrenAsArray, (child) => _.get(child, 'type.displayName', '') === 'VerticalListMenu');
```

Comment:
in the `child-component` util, we have a function called [findElementsByType](https://github.com/appnexus/lucid/blob/master/src/util/child-component.js#L87), that does this same logic but compares the actual types. `findElementsByType(children, [VerticalListMenu])`. i have a branch that does the below logic, with a function called `rejectElementByType` which is almost the same logic.